pkg/frontend/admin_openshiftcluster_upgrade.go

```diff
@@ -5,9 +5,12 @@ package frontend
 
```

Comment:
I don't agree with re-adding this file.  It is copy/pasting complex functionality from putorpatch.  The alternative is simple change to the Geneva Actions codebase.